Documentation/app-operations.md

```diff
@@ -0,0 +1,225 @@
+
+```bash
+rkt app inject <pod-uuid> <image-name/hash/address/registry-URL> <arguments>
+```
+
+## rkt app eject
+Ejects an application image into a running pod, meaning that the leftover resources are removed from the pod.
+
+```bash
+rkt app eject <pod-uuid> --app=<app-name> <arguments>
+```
+
+### Resources Leftover by an Application in Default Stage1 Flavor
+- Rootfs (e.g. `/opt/stage/<app-name>`)
+- Mounts from volumes (e.g. `/opt/stage/<app-name>/<volume-name>`)
+- Mounts related to rkt operations (e.g. `/opt/stage/<app-name>/dev/null`)
+- Systemd service files (e.g. `<app-name>.service` and `reaper-<app-name>.service`)
+- Miscellaneous files (e.g. `/rkt/<app-name>.env`, `/rkt/status...`)
+
+## rkt app start
+Starts an application that was either stopped or injected.
+This operation is idempotent.
+
+```bash
+rkt app start <pod-uuid> --app=<app-name> <arguments>
+```
+
+## rkt app stop
+Stops a running application gracefully (grace is defined in the `app stop` entrypoint section).
+
+```bash
+rkt app stop <pod-uuid> --app=<app-name> <arguments>
+```
+
+## rkt app list
+Lists the applications that are inside a pod, running or stopped.
+
+```bash
+rkt app list <pod-uuid> <arguments>
+```
+
+## rkt app status
+Returns the execution status of application inside a pod.
+
+```bash
+rkt app status <pod-uuid> --app=<app-name> <arguments>
+```
+
+which returns status information for an example mysql application along the lines of:
+
+```bash
+● mysql.service - Application=mysql Image=registry-1.docker.io/library/mysql
+   Loaded: loaded (/usr/lib64/systemd/system/mysql.service; static; vendor preset: enabled)
+      Active: active (running) since Wed 2016-07-09 17:00:50 UTC; 8min 38s ago
+       Main PID: 6 (mysqld)
+          CGroup: /machine.slice/machine-rkt\x2d11efeefb\x2d2e8d\x2d47d0\x2da4ee\x2de1631327acac.scope/system.slice/mysql.service
+                     └─6 mysqld
+
+  Jul 15 17:01:03 rkt-$uuid mysql[6]: [Note]   - '::' resolves to '::';
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Note] Server socket created on IP: '::'.
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Note] InnoDB: Buffer pool(s) load completed at 160713 17:01:03
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Note] Event Scheduler: Loaded 0 events
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: [Note] mysqld: ready for connections.
+  Jul 13 17:01:03 rkt-$uuid mysql[6]: Version: '5.7.13'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
+```
+
+## rkt app exec
```

Comment:
It'll most likely be an alias for `rkt enter`, since the behavior is identical, but we shouldn't be using `app` prefix for the whole api except  `rkt enter`, its a bit confusing